api-documentation.js

```diff
@@ -338,8 +401,55 @@ class ApiDocumentation extends AmfHelperMixin(LitElement) {
     if (old === value) {
       return;
     }
-    this._selectedType = value;
     this.__amfChanged();
+
+    if (!this.handleNavigationEvents) {
+      this._updateServers();
+    }
+
+    this._selectedType = value;
+  }
+
+  get selectedServerValue() {
+    return this._selectedServerValue;
+  }
+
+  set selectedServerValue(value) {
```

Comment:
This getter and setter does nothing. Literally.  Please, define `selectedServerValue` in the `properties` definition. YOu can read more about properties here: https://lit-element.polymer-project.org/guide/properties

Reply:
Got it! I knew this had to be easier 😆